lightbug_http/http/common_response.mojo

```diff
@@ -0,0 +1,45 @@
+fn OK(body: String) -> HTTPResponse:
```

Comment:
Took me a while to think about possible other names for this file but I think `common_response` is good 😄